src/core/tokenStore.js

```diff
@@ -116,28 +112,26 @@ export default function tokenStoreFactory(options) {
          *
          * @returns {Promise<Array>}
          */
-        getIndex: function getIndex() {
-            return this.getTokens().then(function(tokens) {
-                return _.chain(tokens)
-                    .values()
-                    .sort(function(t1, t2) {
-                        return t1.receivedAt - t2.receivedAt;
-                    })
-                    .map('value')
-                    .value();
-            });
+        getIndex() {
+            return this.getTokens()
+                .then(tokens => {
+                    return _.chain(tokens)
```

Comment:
With ES6 I think we can lose lodash:
```
return tokens
.sort((t1, t2) => t1.receivedAt - t2.receivedAt))
.map(token => token.value);
```
Also, the return statement directly inside the function body could be simplified (`tokens => tokens.map(...)`), unless it is stylistically nicer to keep it.

Reply:
Well, `tokens` is not an array. This is a situation where `lodash` is still useful: apply algorithms on a value no matter the data type.